project/BuildPlugin.scala

```diff
@@ -53,17 +55,26 @@ object BuildKeys {
   final val BenchmarkBridgeCompilation = ProjectRef(BenchmarkBridgeProject.build, "compilation")
 
   import sbt.{Test, TestFrameworks, Tests}
-  import sbt.io.syntax.fileToRichFile
+  val buildBase = Keys.baseDirectory in sbt.ThisBuild
   val integrationTestsLocation = Def.settingKey[sbt.File]("Where to find the integration tests")
   val scriptedAddSbtBloop = Def.taskKey[Unit]("Add sbt-bloop to the test projects")
+  val updateHomebrewFormula = Def.taskKey[Unit]("Update Homebrew formula")
   val testSettings: Seq[Def.Setting[_]] = List(
-    integrationTestsLocation := (Keys.baseDirectory in sbt.ThisBuild).value / "integration-tests" / "integration-projects",
+    integrationTestsLocation := buildBase.value / "integration-tests" / "integration-projects",
     Keys.testOptions += Tests.Argument(TestFrameworks.JUnit, "-v", "-a"),
     Keys.libraryDependencies ++= List(
       Dependencies.junit % Test
     ),
   )
 
+  import ohnosequences.sbt.GithubRelease.{keys => GHReleaseKeys}
+  val releaseSettings = Seq(
+    GHReleaseKeys.ghreleaseRepoOrg := "scalacenter",
+    GHReleaseKeys.ghreleaseRepoName := "bloop",
+    GHReleaseKeys.ghreleaseAssets += ReleaseUtils.versionedInstallScript.value,
```

Comment:
This means we're running `versionedInstallScript` twice, one here and one in `updateHomebrewFormula`. Would it be possible to make `versionedInstallScript` a setting instead of a task?

Reply:
I changed it, but now it's a side-effecting setting which is not super cool either...

Reply:
Oh damn... if it's side-effecting, it's better as a task. Can we have it as a cache that you only run once (with some kind of semaphore or sync utility) please? 🙏